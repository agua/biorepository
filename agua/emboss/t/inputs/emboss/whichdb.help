whichdb --help
Search all sequence databases for an entry and retrieve it
Version: EMBOSS:6.5.7.0

   Standard (Mandatory) qualifiers (* if not always prompted):
  [-entry]             string     ID or Accession number (Any string)
*  -outfile            outfile    [*.whichdb] Output file name

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -get                toggle     [N] Retrieve sequences
   -showall            boolean    [N] Show failed attempts

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose

