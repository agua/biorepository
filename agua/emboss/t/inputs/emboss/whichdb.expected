[{
	"value":"",
	"name":"entry",
	"discretion":"required",
	"description":"ID or Accession number (Any string)",
	"argument":"-entry",
	"paramtype":"input",
	"valuetype":"string",
	"category":"string"
}
,
{
	"value":"*.whichdb",
	"name":"outfile",
	"discretion":"required",
	"description":"Output file name",
	"argument":"-outfile",
	"paramtype":"input",
	"valuetype":"file",
	"category":"outfile"
}
,
{
	"value":"N",
	"name":"get",
	"discretion":"optional",
	"description":"Retrieve sequences",
	"argument":"-get",
	"paramtype":"input",
	"valuetype":"string",
	"category":"toggle"
}
,
{
	"value":"N",
	"name":"showall",
	"discretion":"optional",
	"description":"Show failed attempts",
	"argument":"-showall",
	"paramtype":"input",
	"valuetype":"boolean",
	"category":"boolean"
}]